.gitignore

```diff
@@ -110,3 +110,13 @@ conda_decompy/
 docs/build
 docs/doctrees/
 .buildinfo
+
+
+# Our Info
```

Comment:
This comment is useless and the blank lines are unneeded. You could label it "#Database files" or something. NOTE: I won't deny the request for this, just mentioning in case other changes are needed.